core/server/common/src/main/java/alluxio/master/journal/JournalTool.java

```diff
@@ -76,16 +117,17 @@
           "The start log sequence number (inclusive). Set to 0 by default.")
       .addOption("end", true,
           "The end log sequence number (exclusive). Set to +inf by default.")
+      .addOption("inputDir", true,
+          "The input directory to read journal content from.")
```

Comment:
If it's easy to do so, could we add something to tell users more information about `inputDir`, like it is not required and the input dir should be disk directory